src/components/Entry/DomainArchitectures/index.tsx

```diff
@@ -0,0 +1,332 @@
+        entry === 'pfam'
+          ? pfLocation?.coordinates
+          : iprLocation?.coordinates || pfLocation?.coordinates;
+    }
+    return {
+      accession: entry === 'pfam' ? pf : ipr || pf,
+      name: entry === 'pfam' ? namePf : nameIpr || namePf,
+      unintegrated: !ipr,
+      locations,
+    } as ProcessedDomain;
+  });
+  const grouped = domains.reduce(
+    (obj, domain) => {
+      if (obj[domain.accession])
+        obj[domain.accession].locations.push(domain.locations[0]);
+      else
+        obj[domain.accession] = { ...domain, locations: [...domain.locations] };
+      return obj;
+    },
+    {} as Record<string, ProcessedDomain>,
+  );
+  const obj = {
+    length: representative?.length || FAKE_PROTEIN_LENGTH,
+    domains: Object.values(grouped),
+    accessions: domains.map((d) => d.accession),
+  };
+  return obj;
+};
+
+const getMaxLength = (
+  idaResults: Array<{ representative?: { length: number } }>,
+) => {
+  const max = Math.max(
+    ...idaResults.map((result) => result?.representative?.length || 0),
+  );
+  return max === 0 ? FAKE_PROTEIN_LENGTH : 100 * Math.ceil(max / 100);
+};
+
+type Props = {
+  mainAccession?: string;
+  search?: Record<string, string>;
+  highlight?: Array<string>;
+  idaAccessionDB?: string;
+  database: string;
+};
+interface LoadedProps
+  extends Props,
+    LoadDataProps<RootAPIPayload, 'DB'>,
+    LoadDataProps<IDAPayload> {}
+
+export const DomainArchitecturesWithData = ({
+  data,
+  mainAccession,
+  search,
+  dataDB,
+  highlight = [],
+  idaAccessionDB,
+  database,
+}: LoadedProps) => {
+  const { loading, payload, status } = data || {};
+  if (loading || dataDB?.loading) return <Loading />;
+  const edgeCaseText = edgeCases.get(status || 0);
+  if (edgeCaseText)
+    return <EdgeCase text={edgeCaseText} status={status || 0} />;
+
+  if (!payload?.results) return null;
+  const toHighlight =
+    highlight.length === 0 && mainAccession ? [mainAccession] : highlight;
+
+  let messageContent;
+  if (payload.count === 0) {
+    const searchEntries = (search?.ida_search || '').split(',');
+    const invalidEntries: Array<string> = [];
+    searchEntries.forEach((entry) => {
+      const matches = entry.match(new RegExp(/(ipr|pf)/, 'ig'));
+      if (!matches) invalidEntries.push(entry);
+    });
+
+    messageContent = (
+      <div className={css('callout', 'info', 'withicon')}>
+        <b>No Domain architectures found</b>. They are calculated for InterPro
+        entries of type <b>Domain</b>. Please make sure the type is Domain for
+        the accessions you are searching for.
+        {invalidEntries.length > 0 ? (
+          <div style={{ margin: '1rem' }}>
+            Please check if the following are valid <b>InterPro</b> or{' '}
+            <b>Pfam</b> accessions:
+            <ul>
+              {invalidEntries.map((e) => (
+                <li key={e}>{e}</li>
+              ))}
+            </ul>
+          </div>
+        ) : null}
+      </div>
+    );
```

Comment:
```suggestion
    messageContent = (
      <div className={css('callout', 'warning')}>
        No domain architectures found. Domain architectures are calculated for InterPro entries of type Domain.
        Please ensure that the entries you are searching are of type Domain.
      </div>
    );
```

Changes:

- Use a _warning_ callout
- Update message
- Remove the bit with _Please check if the following are valid_, as it shows a list with one empty element (see screenshot)

![image](https://github.com/ProteinsWebTeam/interpro7-client/assets/2097501/7860b1d9-eccd-402b-a4c6-f9d1182288c7)